.ci/validation.sh

```diff
@@ -330,15 +330,24 @@ verify-no-exception-configs)
     --no-clobber \
     https://raw.githubusercontent.com/checkstyle/contribution/master/checkstyle-tester/checks-only-javadoc-error.xml
   MODULES_WITH_EXTERNAL_FILES="Filter|ImportControl"
-  xmlstarlet sel --net --template -m .//module -v "@name" \
-    -n $working_dir/checks-nonjavadoc-error.xml -n $working_dir/checks-only-javadoc-error.xml \
-    | grep -vE $MODULES_WITH_EXTERNAL_FILES | grep -v "^$" \
-    | sort | uniq | sed "s/Check$//" > $working_dir/web.txt
-  xmlstarlet sel --net --template -m .//module -v "@name" -n config/checkstyle_checks.xml \
+  xmlstarlet fo -D \
+    -n $working_dir/checks-nonjavadoc-error.xml \
+    | xmlstarlet sel --net --template -m .//module -n -v "@name" \
+    | grep -vE $MODULES_WITH_EXTERNAL_FILES | grep -v "^$" > $working_dir/temp.txt
+  xmlstarlet fo -D \
+     -n $working_dir/checks-only-javadoc-error.xml \
+    | xmlstarlet sel --net --template -m .//module -n -v "@name" \
+    | grep -vE $MODULES_WITH_EXTERNAL_FILES | grep -v "^$" >> $working_dir/temp.txt
```

Comment:
Changed this to a temp file to combine the 2. The next command wasn't working for me doing it all on 1 file.